manifests/install.pp

```diff
@@ -35,8 +35,7 @@
     source          => $keycloak::download_url,
     creates         => "${keycloak::install_dir}/keycloak-${keycloak::version}/bin",
     cleanup         => true,
-    user            => 'keycloak',
-    group           => 'keycloak',
+    user            => $keycloak::user,
+    group           => $keycloak::user,
```

Comment:
Should probably be `$keycloak::group` for `group`

Reply:
You are right. FIxed it already